ps.go

```diff
@@ -42,12 +42,21 @@ var psCommand = cli.Command{
 			return nil
 		}
 
-		psArgs := context.Args().Get(1)
-		if psArgs == "" {
-			psArgs = "-ef"
+		// [1:] is to remove command name, ex:
+		// context.Args(): [containet_id ps_arg1 ps_arg2 ...]
+		// psArgs:         [ps_arg1 ps_arg2 ...]
+		//
+		psArgs := context.Args()[1:]
+
+		if len(psArgs) > 0 && psArgs[0] == "--" {
```

Comment:
Why is this not implemented in the library itself? Currently Docker is moving to Cobra, so maybe we should do the same if it means we don't have to embed these hacks into our client code.